add-on/panels/js/roomStore.js

```diff
@@ -124,6 +125,13 @@ loop.store = loop.store || {};
     },
 
     /**
+     * Pass a sharedAction.SetOwnDisplayName from chrome to content space.
+     */
+    setOwnDisplayName: function(actionData) {
+      loop.request("SetOwnDisplayName", actionData);
```

Comment:
as the current flow is `panel.jsx` requesting a change of preferences, and then dispatching the action that `roomStore.js` is receiving, just to request another API call, maybe we can change it for panel requesting this new `SetOwnDisplayName` instead, and the API setting the preference right before broadcasting the message, so no changes on roomStore are needed.
Although I'm not strong on this, it seems, at first glance, less messy for the action/messages back and forth.